system_files/shared/usr/libexec/configure-grub.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source /usr/lib/ujust/ujust.sh
+GRUB_STATE="$(sudo grub2-editenv list | grep "^menu_auto_hide=")"
+if [ "$GRUB_STATE" == "menu_auto_hide=0" ] ; then
+    GRUB_STATE="${b}Not Hidden${n}"
+else
+    GRUB_STATE="${b}Hidden${n}"
+fi
+echo "${bold}Grub menu configuration${normal}"
+echo "Grub menu is set to: $GRUB_STATE"
+OPTION=$(Choose "Hide Grub" "Unhide Grub" )
+if [[ "${OPTION,,}" =~ ^hide ]]; then
+    sudo grub2-editenv - set menu_auto_hide=2 boot_success=1
```

Comment:
menu_auto_hide=1 should hide the boot_menu so long as there was a successful shutdown.

Don't set boot_success=1. That variable should only be set when the computer successfully powers down.